docs/zh_cn/04-supported-codebases/mmpretrain.md

```diff
@@ -190,3 +190,4 @@ for label_id, score in result:
 | [MobileOne](https://github.com/open-mmlab/mmpretrain/tree/main/configs/mobileone)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [EfficientNet](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientnet)            |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/conformer)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
+| [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientformer)            |      Y      |      Y       |    Y     |  N   |   ?   |    N     |
```

Comment:
Conformer -> EfficientFormer

Reply:
are ncnn and openvino not supported?

Reply:
> are ncnn and openvino not supported?

I re-test it, openvino is supported but the convert result of ncnn is False.